src/app/@ansyn/core/components/imagery-status/imagery-status.component.ts

```diff
@@ -189,4 +197,20 @@ export class ImageryStatusComponent implements OnInit, OnDestroy {
 	backToWorldView() {
 		this.store$.dispatch(new BackToWorldView({ mapId: this.mapId }));
 	}
+
+	removeOverlay() {
+		if (this.overlay && this.overlay.id) {
+			if (this.isRemoved()) {
+				const newRemovedOverlaysIds = this.removedOverlaysIds.filter((id) => id !== this.overlay.id);
+				this.store$.dispatch(new SetRemovedOverlaysIdsAction({idsToRemove: newRemovedOverlaysIds, resetFirst: true }));
```

Comment:
create an simple action for this action

{ id: overlayId, value: this.isRemoved() (`boolean`) }

Reply:
👍